apps/example/pages/admin/[[...nextadmin]].tsx

```diff
@@ -33,14 +33,34 @@ const options: NextAdminOptions = {
         },
       },
       edit: {
-        display: ["id", "name", "email", "posts", "role", "birthDate"],
+        display: [
+          "id",
+          "name",
+          "email",
+          "posts",
+          "role",
+          "birthDate",
+          "avatar",
+        ],
         fields: {
           email: {
             validate: (email) => email.includes("@") || "Invalid email",
           },
           birthDate: {
             format: "date",
           },
+          avatar: {
+            format: "data-url",
```

Comment:
```suggestion
            format: "data-url",
```

If possible we should have "file" instead of "data-url", it is more explicit for the end user.